### ISSUE TITLE
Add Crisis Support for other English-speaking Countries

### ISSUE THREAD

**Issue by eddielement** [suicidesafetyplan/safetyplan-ios#9]
Right now the Crisis tab of the app only shows up for users in the US. The Android version already supports other countries, we should do the same here! The primary things that need to be customized to the country is the Suicide Hotline Phone # and a number to Text if it exists in the country targeted.

**Comment by pankova** [comment]
Hi! Could you clarify the issue? Should Emergency and Helpline numbers and Crisis Text Line email address be founded? Or is there a list of data to add to the iOS app? If we need to find data, we need to do it for each English-speaking country separately, right?

**Comment by eddielement** [comment]
Yes! The Crisis tab is good for US users already, but for non-US users on a per country basis they'd need: 

1. Emergency Number
2. Crisis Line Number (it often goes by a different name to)
3. (Optional, since very few countries actually have one) - Some sort of text-based service

They're in the Android app, but easiest way is probably to grab them from here: 

https://en.wikipedia.org/wiki/List_of_suicide_crisis_lines

In fact, the US number has recenlty changed (old number still works) from 1-800-273-8255 to 988

For the big 4 english countries: 
- US: Emergency - 911, Helpline "Lifeline" - 988, Textline "Crisis Text Line" - Text HOME to 741741
- UK: Emergency - 999, Helpline "Samaritans" - 116 123, Textline "Shout" - Text SHOUT to 85258
- Australia: Emergency - 000, Lifeline - 13 11 14, no text line. 
- Canada: Emergency - 911, Helpline: 1-833-456-4566, Textline "Crisis Text Line" - Text HOME to 686868
